observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -290,6 +291,42 @@ include::snippets/technology-preview.adoc[]
 <6> Disables verifying the server's certificate chain and host name. The default is `+false+`.
 <7> ServerName indicates the name of the server requested by the client to support virtual hosting.
 
+[id="file-exporter_{context}"]
+== File Exporter
```

Comment:
Certainly out of scope, but all of these sections seem like modules that should be broken out and included in the assembly per mod docs. Might be a thing to look at another time.

Reply:
Thank you. I'm wondering how this will be migrated into our new tooling.

Reply:
my guess would be a bunch of reference topics, but yeah we shall see :sweat_smile: